### PR TITLE
Update .NET cli verion used to build xUnit-Performance

### PR DESCRIPTION
**Pull request by jorive**

### CODE REVIEW

**Comment by jorive** [comment]
@ViktorHofer PTAL

File: .vscode/extensions.json [hunk 1]
```diff
@@ -1,5 +1,7 @@
 {
     "recommendations": [
-        "ms-vscode.csharp"
+        "ms-vscode.csharp",
+        "DavidAnson.vscode-markdownlint",
+        "ms-vscode.csharp",
```
Comment (ViktorHofer) [inline .vscode/extensions.json]: usually we don't include the .vscode directory in git

File: DotNetCliVersion.txt [hunk 2]
```diff
@@ -1 +1 @@
-2.0.0
\ No newline at end of file
+2.1.302
```
Comment (ViktorHofer) [inline DotNetCliVersion.txt]: this value is not picked up by the sh script right? As you hardcoded it there.

File: dotnet-install.cmd [hunk 3]
```diff
@@ -30,25 +28,24 @@ setlocal
...
   if not exist "%DotNet_Path%\dotnet-install.ps1" (
     call :print_error_message Failed to download "%DotNet_Path%\dotnet-install.ps1"
     exit /b 1
   )
 
-  echo Executing dotnet installer script "%DotNet_Path%\dotnet-install.ps1"
-  for %%v in (2.0.0) do (
-    echo Installing dotnet sdk version %%~v
+  call :print_header_message Executing dotnet installer script "%DotNet_Path%\dotnet-install.ps1"
+  for %%v in (%DotNet_Version%) do (
```
Comment (ViktorHofer) [inline dotnet-install.cmd]: we can get rid of the loop here, right?

**Review by ViktorHofer — APPROVED** [review]
LGTM if it builds well.

File: .vscode/extensions.json (hunk 1 above)
Comment (jorive) [reply .vscode/extensions.json]: This file provides suggestions of the extensions to download. I think it is useful, specially for new users.

File: DotNetCliVersion.txt (hunk 2 above)
Comment (jorive) [reply DotNetCliVersion.txt]: This version is used by the `.cmd` and `.sh` files. Ideally, I would consolidate the build scripts into a single script.

File: dotnet-install.cmd (hunk 3 above)
Comment (jorive) [reply dotnet-install.cmd]: Sure.

File: DotNetCliVersion.txt (hunk 2 above)
Comment (ViktorHofer) [reply DotNetCliVersion.txt]: So is the hardcoded version string here needed? https://github.com/Microsoft/xunit-performance/pull/271/files#diff-ba04653bd26665aa2b926dbe2b144e61R21

File: .vscode/extensions.json (hunk 1 above)
Comment (ViktorHofer) [reply .vscode/extensions.json]: fine with me :)

File: DotNetCliVersion.txt (hunk 2 above)
Comment (jorive) [reply DotNetCliVersion.txt]: Good call, it needs to be updated.

**Comment by ViktorHofer** [comment]
No idea why GitHub says that I authored your first commit here 😂. LGTM. I'm happy that repo is now finally again up2date code and infrastructure wise :)

**Comment by jorive** [comment]
Me neither... I'm confused :P

**Comment by ViktorHofer** [comment]
Maybe incrementing the versions to 1.0.0 would really make sense. Not sure...

**Comment by jorive** [comment]
Let's do it with the next source code changes. Maybe adding Code Analysis?

**Comment by ViktorHofer** [comment]
Oooh that sounds fancy 😁

**Pull request merged**